app/res/values-de/strings.xml

```diff
@@ -252,8 +252,8 @@
     <string name="code">Code</string>
     <string name="following_user">Folgen…</string>
     <string name="unfollowing_user">Entfolgen…</string>
-    <string name="starring_repository">Hervorheben…</string>
-    <string name="unstarring_repository">Hervorhebung entfernen…</string>
+    <string name="starring_repository">Favorisiere Repository…</string>
+    <string name="unstarring_repository">Defavorisiere Repository…</string>
```

Comment:
The phrases for unstarring gists and repositories should match.

When ```unstarring_gist``` is translated as "Entferne Gist von Favoriten" ```unstarring_repository``` should be "Entferne Repository von Favoriten". Or ```unstarring_gist``` should be "Defavorisiere Gist"

In my opinion the long forms "Entferne XX von (den) Favoriten" are better